exchange/exchange.go

```diff
@@ -142,34 +145,34 @@ func (e *exchange) getAllBids(ctx context.Context, cleanRequests map[openrtb_ext
 			ae := new(seatResponseExtra)
 			ae.ResponseTimeMillis = int(elapsed / time.Millisecond)
 			// Timing statistics
-			e.m.AdapterMetrics[coreBidder].RequestTimer.UpdateSince(start)
+			e.me.RecordAdapterTime(*blabels[coreBidder], time.Since(start))
 			serr := make([]string, len(err))
 			for i := 0; i < len(err); i++ {
 				serr[i] = err[i].Error()
 				// TODO: #142: for a bidder that return multiple errors, we will log multiple errors for that request
 				// in the metrics. Need to remember that in analyzing the data.
 				switch err[i] {
 				case context.DeadlineExceeded:
-					e.m.AdapterMetrics[coreBidder].TimeoutMeter.Mark(1)
+					blabels[aName].RequestStatus = pbsmetrics.RequestStatusTimeout
```

Comment:
`aName` may be an alias. I think these should log under `coreBidder` so we don't explode the number of metrics we're storing.

Reply:
`blabels[aName]` is a map to a label, the bidder in the lable doesn't have to be `aName` I tired to be sure I used `coreBidder` when setting the bidder name inside of the label itself, but I will review all that code and make sure I only send corebidders as bidder flags.

Reply:
`blabels[aName]` is a bug, it should be `blabels[coreBidder]` I was correct that I always set the bidder name inside the labels as the core bidder, not sure how I ended up with some map references as `aName` instead.